packages/lib-classifier/src/store/subjects/Subject/SubjectLocation/SubjectLocation.js

```diff
@@ -0,0 +1,46 @@
+import { types } from 'mobx-state-tree'
+
+const fileExtensions = {
```

Comment:
With the same constant in `validateSubjectLocations` I'm not sure if it would make sense to export from one place or move `fileExtensions` somewhere to be imported in both places. For consideration, not blocking.

Reply:
I’m not sure if `validateSubjectLocations` is used anywhere either.